examples/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2020, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+# Getting Started - Example 1
+add_executable(iidm4cpp_reader getting-started/example1/main.cpp)
+target_link_libraries(iidm4cpp_reader iidm)
```

Comment:
I don't understand the folder organisation you propose. It means we would have several example in the getting-started folder. What would be the other examples?

I would propose to have only one hierarchy level:
- example1/
- example2/
And a README.md file that explains the purpose of the example.

@gautierbureau What do you think?